src/components/forms/EmailField/EmailField.js

```diff
@@ -11,6 +11,7 @@ EmailField.propTypes = {
   description: PropTypes.node,
   id: PropTypes.string,
   disabled: PropTypes.bool,
+  autocomplete: PropTypes.string,
```

Comment:
```suggestion
  autocomplete: PropTypes.oneOf(['email']),
```
it doesn't make sense to allow other autocomplete strings, does it?

Reply:
Ah check, yeah i guess so. Or at least, for now that is the correct solution. I've changed it 👍